src/main/io/serial_cli.c

```diff
@@ -547,6 +547,10 @@ static const char * const lookupTablePwmProtocol[] = {
 #endif
 };
 
+static const char * const lookupTablePwmPolarity[] = {
+    "POSITIVE", "NEGATIVE"
+};
```

Comment:
I don't really like the use of the words "POSITIVE" and "NEGATIVE" here. I would prefer to use "INVERTED", but not sure what the other word should be.